dandi/models.py

```diff
@@ -525,28 +498,20 @@ class CommonModel(DandiBaseModel):
         nskey="schema",
     )
     studyTarget: Optional[List[Union[str, AnyUrl]]] = Field(
-        None,
-        title="Study Target",
-        description="What the study is related to",
-        nskey="dandi",
-    )
-    license: List[LicenseType] = Field(
-        title="License", description="License of item.", nskey="schema"
+        None, description="What the study is related to", nskey="dandi"
     )
+    license: List[LicenseType] = Field(description="License of item.", nskey="schema")
```

Comment:
@jwodder - then we should not remove the title field that was added to enum types. There are a few of these. 

```
AccessType = create_enum(AccessTypeDict)
RoleType = create_enum(RoleTypeDict)
RelationType = create_enum(RelationTypeDict)
LicenseType = create_enum(LicenseTypeDict)
IdentifierType = create_enum(IdentifierTypeDict)
DigestType = create_enum(DigestTypeDict)
```

Reply:
Oh, you mean the enum *fields*!  No, those titles are autogenerated fine.  It's the enum entries in the `definitions` section that don't get titles.